float64 dist
std_msgs/String flag
